plugins/boltz/swapmachine.go

```diff
@@ -80,7 +83,7 @@ func (s *SwapMachine) FsmSwapSuccess(in FsmIn) FsmOut {
 	if in.MsgCallback != nil {
 		in.MsgCallback(entities.PluginMessage{
 			JobID:      int32(in.GetJobID()),
-			Message:    fmt.Sprintf("Swap %d succeeded", in.GetJobID()),
+			Message:    fmt.Sprintf("Swap %d succeeded (dry run: %v)", in.GetJobID(), in.SwapData.IsDryRun),
```

Comment:
please write different message for dry run. I don't want client to see anything about dry run when executing without it.

`Swap %d succeded`
`Swap %d succeded in dry run mode`